variables.tf

```diff
@@ -49,7 +49,7 @@ variable "alertmanager_port" {
 }
 
 variable "cloudwatch_exporter_port" {
-  default = 9106
+  default = 5000
```

Comment:
Is it worth keeping the 9106 port? Did I miss this in the other PR? I did.  Ugh.